### ISSUE TITLE
v1.100 - ExtraBold fonts - nameID 2 is wrong

### ISSUE THREAD

**Issue by kenmcd** [githubnext/monaspace#210]
In the v1.100 release all the ExtraBold fonts, Regular and Italic, have `Bold` and `Bold Italic` in `nameID 2`.

These should only be `Regular` or `Italic`.

**Comment by idan** [comment]
Investigating!

**Comment by Finii** [comment]
![image](https://github.com/githubnext/monaspace/assets/16012374/790856b3-8131-427a-819b-1fd4b66bd8d0)

I'm not sure about this one (`Bold` flag in fsSelection and macstyle):
At least it sets it consistently:

```console
$ showttf  monaspace-v1.100/fonts/otf/MonaspaceXenon-WideExtraBold.otf 2>/dev/null | grep 'fsSelect|macstyle'
	macstyle=1
	 fsSelection=160
```

While these have it not-set
```console
$ showttf  ../Noto/Sans/NotoSans_Condensed-ExtraBold.ttf 2>/dev/null | grep 'fsSelect|macstyle'
	macstyle=0
	 fsSelection=192
$ showttf  ../JetBrainsMono/NoLigatures/ExtraBold/JetBrainsMonoNL-ExtraBold.ttf 2>/dev/null | grep 'fsSelect|macstyle'
	macstyle=0
	 fsSelection=192
$ showttf /usr/share/fonts/opentype/cantarell/Cantarell-ExtraBold.otf 2>/dev/null | grep 'fsSelect|macstyle' 
	macstyle=0
	 fsSelection=64
$ showttf /usr/share/fonts/truetype/open-sans/OpenSans-ExtraBold.ttf 2>/dev/null | grep 'fsSelect|macstyle' 
	macstyle=0
	 fsSelection=0
```

https://learn.microsoft.com/en-us/typography/opentype/spec/os2#fsselection

**Comment by kenmcd** [comment]
> I'm not sure about this one (`Bold` flag in fsSelection and macstyle):
> At least it sets it consistently

That is a different issue, but a related problem.
Mac-only bad habits vs. cross-platforn best practices.

GF FontBakery will definitely kick out the nameID 2 issue as an error which must be fixed.
Not sure on the fsSelection issue. 
Have to check that.

**Comment by idan** [comment]
Should be fixed in e6f6278955d3c47a77c1bdbf442f60838667cb98! Thank you for the report 🥰
